packages/create-subscription/src/createSubscription.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import React from 'react';
+import warning from 'fbjs/lib/invariant';
+
+export function createSubscription<
+  Property,
+  CreatedSubscription,
+  Value,
+>(config: {|
```

Comment:
`$ReadOnly<{}>` can be used instead of `+`

Reply:
Sure. I don't know if one is better than the other really.